src/components/Login.js

```diff
@@ -46,7 +46,7 @@ class Login extends Component {
     if (result === undefined) {
       Actions.login();
     } else {
-      Actions.main();
+      Actions.generalInfo();
```

Comment:
just letting you know, there is absolute difference where Actions.{name} is same as Action push and never pops which will keep scene stacks growing and result poor performance.